howto/manage-juju-controllers.md

```diff
@@ -39,10 +39,14 @@ For this how-to you will need the following:
 
 ### Prelude
 
-In order for a Juju controller to trust a JIMM controller, the `login-token-refresh-url` config option must be set to a specific URL path that serves JIMM's public key, which is used to verify signed
-requests when they reach the Juju controller.
+There are 2 ways to add controllers to JAAS/JIMM:
+1. Use the traditional `juju bootstrap` command and then register the controller with JAAS.
+2. Use the `jaas` CLI tool to have JIMM bootstrap a controller without any client-side logic,
+allowing for greater automation. 
 
-This can be specified manually when bootstrapping the Juju controller directly or is set automatically when bootstrapping through JIMM
+In order for a Juju controller to trust a JIMM controller, the `login-token-refresh-url` config option must be set to a specific URL path that serves JIMM's public key, which is used to verify signed requests when they reach the Juju controller.
+
+This can be specified manually when bootstrapping the Juju controller directly or is set automatically when bootstrapping through JIMM.
```

Comment:
Unnecessary, imo. I'd just make it clear by giving the actual generic workflow under each of 1 and 2.